client/src/components/overview/addToCart/QuantitySelector.jsx

```diff
@@ -15,6 +15,17 @@ function QuantitySelector({ currentStyle, currentSku, setQuantity }) {
     quantityArray.push(i + 1);
   }
 
+  const customStyles = {
+    control: (provided, state) => ({
+      ...provided,
+      borderBottom: '1px dotted pink',
+      color: state.isSelected ? 'red' : 'blue',
+      padding: 5,
+      margin: 10,
+      width: '12vw',
+    }),
+  };
+
   if (totalNum === 0) {
```

Comment:
Do you think you can combine lines 29-41 and 45-58 into one conditional return statement? I think it would look cleaner.